### PR TITLE
Remove gql

### PR DESCRIPTION
**Pull request by johnbarhorst**
GraphQL, while fun to learn and use, does not seem to fit with Next.js very easily. Having one central endpoint for the api feels like it goes against the individual api route system built into Next. Also, the SSG/SSR aspects of Next seem designed around having individual api routes built into the files that use them. Adding GQL to those sorts of pages involve creating wrappers and shipping more code.

TLDR: More headache than it was worth to me at the time.

### CODE REVIEW

**Comment by johnbarhorst** [comment]
Another error, caught by Vercel! Thanks Vercel
